content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -88,7 +88,7 @@ different Kubernetes components.
 | `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | |
 | `DevicePlugins` | `false` | Alpha | 1.8 | 1.9 |
 | `DevicePlugins` | `true` | Beta | 1.10 | |
-| `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.20 |
+| `DisableAcceleratorUsageMetrics` | `false` | Beta | 1.20 | 1.22 |
```

Comment:
It's false by default?

Reply:
Good catch! Thanks for picking it up :) !